Dockerfile

```diff
@@ -6,9 +6,9 @@ MAINTAINER Leigh McCulloch
 # Upload Unison for building
 COPY container /
 
-# Allow setting these on build, set defaults to that of latest.
-ARG UNISON_VERSION=2.48.3
-ARG OCAML_VERSION=4.02
+# Allow setting these on build
+ARG UNISON_VERSION=${UNISON_VERSION}
```

Comment:
Wouldn't it be better to leave these at sane defaults so that this is still a valid Dockerfile syntax-wise and then use something like:
```
sed -e "s/^(ARG\sUNISON_VERSION=)(([0-9]{1}).([0-9]{2}.([0-9]{1})))/2.48.3/"   -e "s/^(ARG\sOCAML_VERSION=)(([0-9]{1}).([0-9]{2})))/4.02/" Dockerfile > Dockerfile-unison2.48.3-ocaml4.02
```